.circleci/config.yml

```diff
@@ -1,5 +1,13 @@
 version: 2.1
 
+parameters:
+  xlarge:
+    type: string
+    default: anf/xlarge
```

Comment:
@robwafle can you modify to be just `xlarge` on both (here and on line 9) please?

Reply:
d73667cc1